concourse/scripts/spec/tag-release_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'fileutils'
+
+prepare_aruba_config()
+
+RSpec.describe "tag-release.sh", :type => :aruba do
+  before(:each) do
+    prepare_ssh_keys()
+    config_git_email_and_name()
+    setup_cloned_repository()
+    clone_repository("paas-cf")
+
+    set_environment_variable 'GIT_REPO_URL', File.join(@workdir, "origin_repo")
+  end
+
+  context("when I checkout the before last tag of 'previous-*'") do
+    before(:each) do
+      bash_c("cd paas-cf && git checkout previous-0.0.2")
+    end
+
+
+    it("should promote the same tag version for 'next-*'") do
+      run("./tag-release.sh next- test_aws_account test_env \"previous-*\"")
+      expect(last_command_started).to have_exit_status(0)
+
+      bash_c_quiet("cd paas-cf && echo a && git tag -l \"next-*\" --sort version:refname | tail -n 1")
```

Comment:
Would this be more succinct if it asserted that `previous-0.0.2` and `next-0.0.2` point at the same commit?